client/components/Editor/commands/rtl.ts

```diff
@@ -0,0 +1,63 @@
+import { Node } from 'prosemirror-model';
+import { EditorState } from 'prosemirror-state';
+
+import { Dispatch } from './types';
+import { createCommandSpec } from './util';
+
+type NodePos = { node: Node; pos: number };
+const directionAttr = 'rtl';
+
+// or type is attr?
+const getDirectionValue = (direction: string, nodes: NodePos[]) => {
+	return [...nodes][0].node.attrs.rtl === direction ? null : direction;
+};
+
+// what nodes able to be rtl, not sure, how to check?
+
+const isRtlTarget = (node: Node) => {
+	const { spec } = node.type;
+	return spec.attrs && directionAttr in spec.attrs;
+};
+
+// nodes could be in ltr or rtl
+const getDirectionNodes = (state: EditorState): NodePos[] => {
```

Comment:
Since you introduced the nicely precise "RTL target" terminology in the function above, I'd stick with it, and name this function `getRtlTargetNodes` or something.